entity/Scalar.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.android_homework_1.entity
+
+
+class Scalar constructor() : Var() {

```

Comment:
такой конструктор можно убрать если тут не используются модификаторы или аннотации
```suggestion
class Scalar : Var() {

```